be/src/exprs/block_bloom_filter_impl.cc

```diff
@@ -69,8 +72,12 @@ Status BlockBloomFilter::init_internal(const int log_space_bytes, uint32_t hash_
 
     const size_t alloc_size = directory_size();
     close(); // Ensure that any previously allocated memory for directory_ is released.
-    _mem_holder.reset(new char[alloc_size]);
-    _directory = (Bucket*)_mem_holder.get();
+    DCHECK(_directory == nullptr);
+    int rc = posix_memalign((void**)&_directory, 32, alloc_size);
```

Comment:
It seems that `posix_memalign` still allocates memory with `TCMalloc`, not `jemalloc`